tests/safeds/ml/classical/classification/test_support_vector_machine.py

```diff
@@ -11,17 +11,69 @@ def training_set() -> TaggedTable:
 
 class TestC:
     def test_should_be_passed_to_fitted_model(self, training_set: TaggedTable) -> None:
-        fitted_model = SupportVectorMachine(c=2).fit(training_set=training_set)
+        kernel = SupportVectorMachine.Kernel.Linear()
+        fitted_model = SupportVectorMachine(c=2, kernel=kernel).fit(training_set=training_set)
         assert fitted_model.c == 2
+        assert isinstance(fitted_model.kernel, SupportVectorMachine.Kernel.Linear)
```

Comment:
remove this line (see reason above)

Reply:
Better to create a duplicate method for TestKernel that tests only the kernel.